solution-base/kafka/charts/kafka-operator/Chart.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+name: kafka-operator
+version: 0.2.18
+description: kafka-operator manages Kafka deployments on Kubernetes
+sources:
+  - https://github.com/banzaicloud/kafka-operator
+appVersion: v0.11.2
```

Comment:
I can see that you made the choice of committing the whole chart in here, this will avoid us some trouble in the future because we're not downloading it from the internet every time, which is nice.

Now I would consider to make the task of "upgrading" this chart easier, standardize. meaning, for this chart and the next ones that we are going to pick here, if any, could be nice to have a helper script, makefile, (whatever fits) that does the right `helm fetch kafka-operator --version $VERSION && tar -xf` command. In order to:
* Track where did this chart came from
* How did we retrieved it
* How do we make the update if needed

I also don't think it's required to do in this PR. If you have the time, great, if not, let me know what you think and we can create a ticket for it.

Reply:
will create a script to pull and apply custom patches